.github/workflows/ci.yml

```diff
@@ -25,22 +25,17 @@ jobs:
       - name: Svelte check
         run: cd frontend && PUBLIC_AMPLITUDE_API_KEY="" npm run check
 
-  format-backend:
+  backend:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: psf/black@stable
```

Comment:
ooc: why are we doing this?

Reply:
ruff is 10-100x faster and then we only need one package to rule them all:

https://twitter.com/tiangolo/status/1716916271793750027